adobe_vipm/flows/migration.py

```diff
@@ -159,110 +159,138 @@ def populate_offers_for_transfer(product_id, transfer, transfer_preview):
     create_offers(product_id, offers)
 
 
+def handle_preview_error(transfer, api_err):
```

Comment:
What this function is doing - updating the transfer object properly according to the api_err. And I didn't get why do you need to return true/false here